lib/matplotlib/backend_bases.py

```diff
@@ -208,7 +208,7 @@ def draw_markers(self, gc, marker_path, marker_trans, path,
     def draw_path_collection(self, gc, master_transform, paths, all_transforms,
                              offsets, offset_trans, facecolors, edgecolors,
                              linewidths, linestyles, antialiaseds, urls,
-                             offset_position):
+                             offset_position, hatchcolors=None):
```

Comment:
Make this keyword-only (everywhere throughout the PR) so that if later we decide to add e.g. a `hatches` kwarg it can go before hatchcolors?